scripts/install_third_party.sh

```diff
@@ -94,9 +94,14 @@ if [ ! "$NO_SKULPT" -a ! -d "$THIRD_PARTY_DIR/static/skulpt-0.10.0" ]; then
 
     # The Skulpt setup function needs to be tweaked. It fails without certain
     # third party commands. These are only used for unit tests and generating
-    # documentation and are not necessary when building Skulpt.
+    # documentation and are not necessary when building Skulpt. Some machines
+    # also have problems running the dist tests, so the second two lines
+    # disable those checks to fix behaviors for those environments, and because
+    # historically these tests don't seem to fail or catch anything.
     sed -e "s/ret = test()/ret = 0/" $TOOLS_DIR/skulpt-0.10.0/skulpt/skulpt.py |\
```

Comment:
Can you add line comments to each of these four lines explaining exactly what each does?

Reply:
Done.